modules/install-sno-ibm-z-lpar.adoc

```diff
@@ -0,0 +1,206 @@
+// This is included in the following assemblies:
+//
+// installing_sno/install-sno-installing-sno.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-sno-on-ibm-z-lpar_{context}"]
+= Installing {sno} in an LPAR on {ibm-z-title} and {ibm-linuxone-title}
+
+.Prerequisites
+
+* If you are deploying a single-node cluster there are zero compute nodes; the Ingress Controller pods run on the control plane nodes. In single-node cluster deployments, you must configure your application ingress load balancer to route HTTP and HTTPS traffic to the control plane nodes. See the _Load balancing requirements for user-provisioned infrastructure_ section for more information.
+* When you create the Kubernetes manifest files, ensure that the `mastersSchedulable` parameter in the `<installation_directory>/manifests/cluster-scheduler-02-config.yml` file is set to `true`, as shown in the following `spec` stanza:
++
+[source,yaml]
+----
+spec:
+  mastersSchedulable: false
```

Comment:
Shouldn't this value be shown as true? We are telling the users to set the value to true

Reply:
True 👍